pkg/csi/service/driver.go

```diff
@@ -47,7 +47,6 @@ var (
 	// CO agnostic orchestrator for the controller as well as node containers.
 	COInitParams  interface{}
 	clusterFlavor = defaultClusterFlavor
-	cfgPath       = cnsconfig.DefaultCloudConfigPath
```

Comment:
why do we remove this line?

```
cfgPath = cnsconfig.DefaultCloudConfigPath
```

Reply:
this was done to resolve linter issue raised.